### ISSUE TITLE
Coba

### ISSUE THREAD

**Issue by yosiaoyos** [yosiaoyos/bronson#1]
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Starter Template for Bootstrap</title>

    
    <link href="http://getbootstrap.com/dist/css/bootstrap.min.css" rel="stylesheet">

    
    <link href="http://getbootstrap.com/assets/css/ie10-viewport-bug-workaround.css" rel="stylesheet">

    
    <link href="http://getbootstrap.com/examples/starter-template/starter-template.css" rel="stylesheet">

    
    
    <script src="http://getbootstrap.com/assets/js/ie-emulation-modes-warning.js"></script>

    
    
  </head>

  <body>

    <nav class="navbar navbar-inverse navbar-fixed-top">
      <div class="container">
        <div class="navbar-header">
          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand" href="#">Website Survey</a>
        </div>
        <div id="navbar" class="collapse navbar-collapse">
          <ul class="nav navbar-nav">
            <li class="active"><a href="#">Home</a></li>
            <li><a href="#about">About</a></li>
            <li><a href="#contact">Contact</a></li>
          </ul>
        </div>
      </div>
    </nav>

    <div class="container">
    	<div class="starter-template">
        	<h1>Welcome to website survey</h1>
    	</div>
    </div>
    <div class="container" style="background-color:#f0f0f0; padding: 30px;">
    	<form action="" method="POST">
    		<div class="col-md-12" style="margin-top:20px; margin-bottom:20px;">
    			<strong>1. Lorem ipsum dolor sit amet consectetur</strong>
    			<div class="col-md-12" style="margin-top:20px;">
    				<input type="radio" name="satu"> Test
    			</div>
    			<div class="col-md-12" style="margin-top:20px;">
    				<input type="radio" name="satu"> Test
    			</div>
    		</div>
    		<div class="col-md-12" style="margin-top:20px; margin-bottom:20px;">
    			<strong>2. Lorem ipsum dolor sit amet consectetur</strong>
    			<div class="col-md-12" style="margin-top:20px;">
    				<input type="radio" name="dua"> Test
    			</div>
    			<div class="col-md-12" style="margin-top:20px;">
    				<input type="radio" name="dua"> Test
    			</div>
    		</div>
    	</form>
    </div>

    
    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
    <script>window.jQuery || document.write('<script src="http://getbootstrap.com/assets/js/vendor/jquery.min.js"><\/script>')</script>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
    
    <script src="http://getbootstrap.com/assets/js/ie10-viewport-bug-workaround.js"></script>
  </body>
</html>
